packages/ember-metal/lib/tracked.ts

```diff
@@ -96,12 +96,13 @@ class Tracker {
 
   @param dependencies Optional dependents to be tracked.
  */
-export function tracked(
-  _target: object,
-  key: string,
-  descriptor: PropertyDescriptor
-): PropertyDescriptor {
-  if ('value' in descriptor) {
+export function tracked(...dependencies: string[]): MethodDecorator;
+export function tracked(target: any, key: any): any;
```

Comment:
I believe these can be typed as `unknown` and `PropertyKey`, respectively.

Reply:
Fixed